irobotclient/test/test_full_pipeline.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+from unittest.mock import MagicMock
+
+import argparse
+import os
+import time
+
+import irobotclient.main
+
+"""
+Likely need to mock request, response, time.sleep.
+"""
+
+
+class TestPipeline(unittest.TestCase):
+
```

Comment:
I've renamed the module and class to be more clear and added brief descriptions to each of the test case classes.